app/routes/index.js

```diff
@@ -0,0 +1,14 @@
+import Route from '@ember/routing/route';
+import ENV from 'website-my/config/environment';
+
+const API_BASE_URL = ENV.BASE_API_URL;
+export default class IndexRoute extends Route {
+  model = async () => {
+    const defaultStatus = 'active';
+    const response = await fetch(`${API_BASE_URL}/users/self`, {
+      credentials: 'include',
+    });
+    const userData = await response.json();
+    return userData.status ?? defaultStatus;
```

Comment:
What if the user is NOT logged in? Are we not handling the case?
You can take a look at how `signup` is handling that

Reply:
Yes, we are not handling that case. Will look into it and update.